src/x509.c

```diff
@@ -5782,16 +5802,10 @@ static int X509PrintName(WOLFSSL_BIO* bio, WOLFSSL_X509_NAME* name,
 /* human readable print out of x509 version
  * return WOLFSSL_SUCCESS on success
  */
-static int _wolfSSL_X509_version_print(WOLFSSL_BIO* bio, WOLFSSL_X509* x509,
-        int indent)
+static int X509PrintVersion_ex(WOLFSSL_BIO* bio, int version, int indent)
```

Comment:
No need for _ex version of this function, it's internal only so the names can change

Reply:
Makes sense. Renamed.